### ISSUE TITLE
tastypie warning from deprecated method override_urls

### ISSUE THREAD

**Issue by radinamatic** [learningequality/ka-lite#3822]
I just cloned my repo from scratch in effort to get rid of some persistent strange errors on the previous one. Everything runs smoothly now, but I got this error on starting the server. 

```
INFO:kalite:Preloading topic tree data for language en.
INFO:kalite:Copying static media...
D:\github15\KA\ka-lite\python-packages\tastypie\resources.py:328: UserWarning: 'override_urls' is a
deprecated method & will be removed by v1.0.0. Please rename your method to ``prepend_urls``.
  warnings.warn("'override_urls' is a deprecated method & will be removed by v1.0.0. Please rename your method to ``prepend_urls``.")
```

No idea if it's important, so close up if nothing needs to be fixed!

**Comment by MCGallaspy** [comment]
It's a warning, so not urgent. But still important to address.

**Comment by benjaoming** [comment]
I'm closing this, because it won't be an issue once unbundled: #3403 

So the true concern is: Get hard-copied packages managed through requirements.txt and we can be up to speed with the rest of the world's software releases :)

**Comment by benjaoming** [comment]
Now tracked in: #3825

**Comment by MCGallaspy** [comment]
Are you sure unbundling tastypie will resolve this issue? It looks like *we're* using a deprecated function in our codebase, and tastypie is throwing the warning.

**Comment by benjaoming** [comment]
Oops, I didn't read the warning correctly, I thought it was coming from something tastypie accessed wrongly.

**Comment by benjaoming** [comment]
I can recommend this blog post written by a sheer genius:

http://overtag.dk/wordpress/2013/03/tracking-down-warnings-in-django/

**Comment by jamalex** [comment]
It *definitely* won't be an issue once @rtibbles and I get rid of tastypie *entirely*.

![](http://dontpkethebear.com/wp-content/uploads/2013/05/sloth-soon-meme1.jpg)

**Comment by rtibbles** [comment]
Punting this to 0.15 as part of the TastyPie off.

**Comment by jamalex** [comment]
Since it gets shown repeatedly to end users, e.g. every time you start the server, a quick fix for this (e.g. ignoring the warning) could be good.

    INFO:kalite:Preloading content data for language en.
    INFO:kalite:Preloading topic tree data for language en.
    INFO:kalite:Copying static media...
    WARNING:py.warnings:/home/jamalex/fle/ka-lite/python-packages/tastypie/resources.py:328: UserWarning: 'override_urls' is a deprecated method & will be removed by v1.0.0. Please rename your method to ``prepend_urls``.
      warnings.warn("'override_urls' is a deprecated method & will be removed by v1.0.0. Please rename your method to ``prepend_urls``.")

    To access KA Lite from another connected computer, try the following address(es):
        http://192.168.1.232:8008/
    To access KA Lite from this machine, try the following address:
        http://127.0.0.1:8008/

    Going to daemon mode, logging to /home/jamalex/.kalite/server.log

**Comment by esander91** [comment]
I found your project through issuehub.io, and hoping to contribute! I can take a look at this issue.

**Comment by benjaoming** [comment]
Haha, I love this issue :) So much talk, then a one-line change :D

Nice work, @esander91 and thanks for jumping in -- is this really the only place in our codebase where we use a deprecated method name?

**Comment by esander91** [comment]
I know! I expected to spend a lot more time digging through the code to find the problem. This is the only deprecated method error warning I get, but I'll keep an eye out for anything else.

**Comment by benjaoming** [comment]
Looked through the codebase, it's the only occurrence alright :)

**Comment by benjaoming** [comment]
Fixed in #4567!
